frontends/web/src/routes/account/account.jsx

```diff
@@ -123,8 +124,12 @@ export default class Account extends Component {
                 connected: !status.includes('offlineMode'),
                 determiningStatus: false,
             };
-            if (!status.initialized && !status.includes('accountDisabled')) {
+            if (!state.initialized && !status.includes('accountDisabled')) {
```

Comment:
Was this again a bug that TypeScript would have prevented (if used correctly)? 😮

Reply:
absolutely!